base/system/chkdsk/lang/ro-RO.rc

```diff
@@ -0,0 +1,38 @@
+/*
+ * PROJECT:     Check Disk utility
+ * LICENSE:     See COPYING in the top level directory
+ * PURPOSE:     Romanian resource file
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+ 
+LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "\n\
+Uz: %s [drive:] [-F] [-V] [-R] [-C] \n\n\
+[drive:]    Specifică modul pilot pentru control.\n\
+-F          Rezolvă erorile discului.\n\
+-V          Vizualizează calea tuturor fișierelor de pe disc.\n\
+-R          Găsește sectoarele defecte și recuperează informațiile care pot fi citite.\n\
+-C          Controlează driver-ul doar dacă este necesar.\n\
```

Comment:
I did not comment previously also for this line but it attracts the same kind of objections for its use of "driver" term.
You probably wanted to say "Controlează unitatea de stocare doar dacă este necesar".